wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -903,4 +921,124 @@ public static function setIndianStateOptions(string $entity, string $field, arra
 
   }
 
+  /**
+   * This hook is called after the database write on a custom table.
+   *
+   * @param string $op
+   *   The type of operation being performed.
+   * @param string $objectName
+   *   The custom group ID.
+   * @param int $objectId
+   *   The entityID of the row in the custom table.
+   * @param object $objectRef
+   *   The parameters that were sent into the calling function.
+   */
+  public static function mailNotificationToMmt($op, $groupID, $entityID, &$params) {
+    if ($op !== 'create') {
+      return;
+    }
+
+    if (!($goonjField = self::findOfficeId($params))) {
+      return;
+    }
+
+    $goonjFieldId = $goonjField['value'];
+    $vehicleDispatchId = $goonjField['entity_id'];
+
+    $collectionSourceVehicleDispatch = EckEntity::get('Collection_Source_Vehicle_Dispatch', FALSE)
+      ->addSelect('Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id')
+      ->addWhere('id', '=', $vehicleDispatchId)
+      ->execute()->first();
+
+    $collectionCampId = $collectionSourceVehicleDispatch['Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id'];
+
+    $coordinators = Relationship::get(FALSE)
+      ->addWhere('contact_id_b', '=', $goonjFieldId)
+      ->addWhere('relationship_type_id:name', '=', self::MATERIAL_RELATIONSHIP_TYPE_NAME)
+      ->addWhere('is_current', '=', TRUE)
+      ->execute()->first();
+
+    $mmtId = $coordinators['contact_id_a'];
+
+    if (empty($mmtId)) {
+      return;
+    }
+
+    $email = Email::get(FALSE)
+      ->addSelect('email', 'contact_id.display_name')
+      ->addWhere('contact_id', '=', $mmtId)
+      ->execute()->single();
+
+    $mmtEmail = $email['email'];
+    $contactName = $email['contact_id.display_name'];
+
+    // Email to material management team member.
+    $mailParams = [
+      'subject' => 'New Entry For Matrial Dispatch Notification',
+      'from' => 'urban.ops@goonj.org',
+      'toEmail' => $mmtEmail,
+      'replyTo' => 'urban.ops@goonj.org',
+      'html' => self::goonjcustom_material_management_email_html($mmtId, $contactName, $collectionCampId),
+        // 'messageTemplateID' => 76, // Uncomment if using a message template
+    ];
+    $result = \CRM_Utils_Mail::send($mailParams);
```

Comment:
`$result` seems to be an unused variable so we can remove it maybe.
```suggestion
    \CRM_Utils_Mail::send($mailParams);
```

Or if you want you can add use it to log some important information obtained from `$result`